### PR TITLE
base.yaml updates for openSUSE (4 of 6)

### PR DESCRIPTION
**Pull request by neotinker**
This is a continuation of the base.yaml updates for openSUSE
#29494 #29640 #29641

These keys are not necessarily related to each other. I'm just processing these commits alphabetically and breaking them up into groups of approx. 10.

https://software.opensuse.org/package/lapack
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/lapack/standard
https://software.opensuse.org/package/libOgreMain1_9_0
https://software.opensuse.org/package/libOgreOverlay1_9_0
https://software.opensuse.org/package/libOgrePaging1_9_0
https://software.opensuse.org/package/libOgreProperty1_9_0
https://software.opensuse.org/package/libOgreRTShaderSystem1_9_0
https://software.opensuse.org/package/libOgreTerrain1_9_0
https://software.opensuse.org/package/libOgreVolume1_9_0
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/ogre/standard
https://software.opensuse.org/package/opencv
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/opencv/standard
https://software.opensuse.org/package/OpenSceneGraph
https://software.opensuse.org/package/libOpenSceneGraph158
https://software.opensuse.org/package/libOpenThreads21
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/OpenSceneGraph/standard
https://software.opensuse.org/package/libpng16
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/libpng16/standard
https://software.opensuse.org/package/libQt5Network5
https://software.opensuse.org/package/libQt5OpenGL5
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2:Update/patchinfo.15202/standard
https://software.opensuse.org/package/libqt5-qtsvg
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/libqt5-qtsvg/standard
https://software.opensuse.org/package/libsensors4
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2/sensors/standard
https://software.opensuse.org/package/libopenssl1_1
https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2:Update/patchinfo.15927/standard

### CODE REVIEW

File: rosdep/base.yaml [hunk 1]
```diff
@@ -4774,6 +4781,7 @@ libssl-dev:
   gentoo: [dev-libs/openssl]
   nixos: [openssl]
   openembedded: [openssl@openembedded-core]
+  opensuse: [libopenssl-devel]
```
Comment (mabelzhang) [inline rosdep/base.yaml:4784]: I think this is the link for `libopenssl-devel` https://build.opensuse.org/package/show/openSUSE%3ALeap%3A15.2/openssl
The .spec file says
```
%package -n libopenssl-devel
```

The original `openssl-1_1` link seems to want the `-1_1` suffix, as the .spec file says
```
%package -n libopenssl-1_1-devel
```

**Review by mabelzhang — APPROVED** [review]

File: rosdep/base.yaml (hunk 1 above)
Comment (tfoote) [inline rosdep/base.yaml:4784]: How does this get tanslaged through https://build.opensuse.org/package/binaries/openSUSE:Leap:15.2:Update/patchinfo.15927/standard ?

I think I see libopenssl1_1-devel

Comment (tfoote) [reply rosdep/base.yaml:4784]: Ahh, good find @mabelzhang

Comment (neotinker) [reply rosdep/base.yaml:4784]: You can build a libopenssl-devel but they dont by default. libopenssl-devel is a meta package that pulls in the correct version of libopenssl devel. 

For openSUSE 15.2, that would be "libopenssl-1_1-devel". I figured that "libopenssl-devel" would be more future proof than "libopenssl-1_1-devel"

```neotinker@ros-15_2:~> zypper info --requires libopenssl-devel
Repository 'Main Update Repository' is out-of-date. You can run 'zypper refresh' as root to update it.
Repository 'Update Repository (Non-Oss)' is out-of-date. You can run 'zypper refresh' as root to update it.
Loading repository data...
Reading installed packages...


Information for package libopenssl-devel:
-----------------------------------------
Repository     : Main Repository
Name           : libopenssl-devel
Version        : 1.1.1d-lp152.1.1
Arch           : noarch
Vendor         : openSUSE
Installed Size : 246 B
Installed      : No
Status         : not installed
Source package : openssl-1.1.1d-lp152.1.1.src
Summary        : Include Files and Libraries mandatory for Development
Description    : 
    This package contains all necessary include files and libraries needed
    to develop applications that require these.
Requires       : [4]
    pkgconfig
    libopenssl1_1 = 1.1.1d
    libopenssl-1_1-devel = 1.1.1d
    openssl = 1.1.1d

neotinker@ros-15_2:~>

Comment (mabelzhang) [reply rosdep/base.yaml:4784]: Thanks for the explanation!

**Pull request merged**